samples/dynamic-headers/custom-visual/src/visual.ts

```diff
@@ -0,0 +1,127 @@
+"use strict";
+
+import "core-js/stable";
+import "./../style/visual.less";
+import powerbi from "powerbi-visuals-api";
+import VisualConstructorOptions = powerbi.extensibility.visual.VisualConstructorOptions;
+import VisualUpdateOptions = powerbi.extensibility.visual.VisualUpdateOptions;
+import IVisual = powerbi.extensibility.visual.IVisual;
+import EnumerateVisualObjectInstancesOptions = powerbi.EnumerateVisualObjectInstancesOptions;
+import VisualObjectInstance = powerbi.VisualObjectInstance;
+import DataView = powerbi.DataView;
+import VisualObjectInstanceEnumerationObject = powerbi.VisualObjectInstanceEnumerationObject;
+import Tabulator from 'tabulator-tables';
+import { VisualSettings } from "./settings";
+
+export class Visual implements IVisual {
+    private target: HTMLElement;
+    private settings: VisualSettings;
+    private dynamic_table_html: HTMLElement;
+    private error_message_html: HTMLElement;
+
+    constructor(options: VisualConstructorOptions) {  
+        this.target = options.element;
+        
+        if (document) {
+            this.dynamic_table_html = document.createElement("div");
+            this.dynamic_table_html.setAttribute("id", "dynamic-header-table");
+
+            this.error_message_html = document.createElement("p"); 
+            this.error_message_html.style.display = "none";
+
+            this.target.appendChild(this.dynamic_table_html);
+            this.target.appendChild(this.error_message_html);
+        }
+    }
+ 
+    public update(options: VisualUpdateOptions) {
+        var tableColumns = options.dataViews[0].table.columns;        
+        var tableRows = options.dataViews[0].table.rows;
+
+        // Rewrite columns index based on their order
+        tableColumns.forEach( (tableColumn, index) =>  tableColumn.index = index );
```

Comment:
NIT: Review extra spaces after `(`, before `)` and after `=> `

It should look like this:
```
tableColumns.forEach((tableColumn, index) => tableColumn.index = index);
```

Please check if there are other places with similar issues.